### DISCUSSION TITLE
arrays of web audio nodes

### DISCUSSION THREAD

**Discussion by artofmus** [cables-gl/cables_docs#571] [Answers & Questions]
Hello! Is there any way to create operators to exchange arrays of web audio nodes instead of one by one to manage multiple audio objects simultaneously, for example to bind them to an array of geometric objects (something like MeshInstancer only for audio nodes) ? I tried to use object inputs and outputs for this, but it didn't work. Any thoughts on this? Thanks!

**Comment by steam0r** [comment]
hey,

does https://cables.local/op/Ops.WebAudio.Mixer maybe work for your usecase?

**Reply by artofmus** [reply]
Thank you! But no, I meant to transmit an array of audio streams over one wire in the same way as happens with ordinary data arrays.
